EasyCommands/BlockHandlers/BlockHandlers.cs

```diff
@@ -10,6 +10,11 @@
 
 namespace IngameScript {
     partial class Program {
+        public List<IMyTerminalBlock> blockCache = NewList<IMyTerminalBlock>();
+        public List<IMyBlockGroup> groupCache = NewList<IMyBlockGroup>();
+        public bool invalidBlockCache = true;
+        public bool invalidGroupCache = true;
```

Comment:
you *might* be able to get away with a null check against blockCache/groupCache instead?

Reply:
no